vignettes/corrected-se.Rmd

```diff
@@ -84,14 +84,14 @@ ev <- block_diag(attr(fs1, which = "fsT"),
                  attr(fs2, which = "fsT"))
 tspa_fit <- tspa(model = "dem60 ~ ind60",
                  data = fs_dat,
-                 cross_loadings = ld,
-                 vc = ev)
+                 fsL = ld,
+                 fsT = ev)
 # Unadjusted covariance
-vcov(tspa_fit)
+fsTov(tspa_fit)
```

Comment:
@winniewytse some of the names that contain `vc` (e.g., `vcov`) have been incorrectly changed. Could you fix those and run a check? Thanks.